src/Microsoft.Azure.WebJobs.Host/JobHostConfiguration.cs

```diff
@@ -20,12 +21,14 @@ public sealed class JobHostConfiguration : IServiceProvider
     {
         private static readonly IConsoleProvider ConsoleProvider = new DefaultConsoleProvider();
 
+        // In some legacy cases, JobHostConfiguration specifically exposes properties on the derived object 
+        // (and not on the core interface in the _services collection).  
```

Comment:
I don't understand this comment at all - I wouldn't classify the below config types as legacy if that is your meaning. Can you just remove this comment? It may cause more confusion than help.